google/cloud/firestore_v1/_helpers.py

```diff
@@ -139,14 +139,20 @@ def verify_path(path, is_collection) -> None:
     """
     num_elements = len(path)
     if num_elements == 0:
-        raise ValueError("Document or collection path cannot be empty")
+        raise ValueError(
+            f"Document or collection path cannot be empty. Encountered {path}"
+        )
 
     if is_collection:
         if num_elements % 2 == 0:
-            raise ValueError("A collection must have an odd number of path elements")
+            raise ValueError(
+                f"A collection must have an odd number of path elements. Encountered {path}"
+            )
     else:
         if num_elements % 2 == 1:
-            raise ValueError("A document must have an even number of path elements")
+            raise ValueError(
+                f"A document must have an even number of path elements. Encountered {path}"
```

Comment:
Is this left over from testing? Do we think that customers will find this more helpful. there is a nonzero chance folks are relying on these messages is all, but I wouldn't recommend doing that anyways so I could be convinced to add this I think?

Reply:
Interesting thought. They are left over from development, but I left them in for the improved clarity. That said, I don't want to take on resolving exception message checking conflicts with this, so I'll revert.

Reply:
I don't think specific message texts are part of the API:  customers who are relying on them could get broken,  e.g., by minor Python version changes as well.